packages/xs-extra/xapi-nbd.master/opam

```diff
@@ -24,7 +24,7 @@ depends: [
   "sexplib"
   "mirage-block-unix"
   "io-page"
-  "mirage"
+  "mirage" {< "3.0.0"}
```

Comment:
Please use https://github.com/ocaml/opam-repository/blob/master/packages/nbd/nbd.2.1.3/opam for nbd, I forgot to push the update to xs-opam